docs/sources/tempo/configuration/_index.md

```diff
@@ -209,6 +209,11 @@ For more information on configuration options, see [here](https://github.com/gra
 
 The ingester is responsible for batching up traces and pushing them to [TempoDB](#storage).
 
+While a trace is considered live, additional spans can be added.
```

Comment:
It makes it seem as if spans can't be added if the trace isn't live. Wether a trace is live in the ingester is an implementation detail that I'm not sure if it's of a lot of value here.

I think it's better suited when talking about per-tenant limits.

Reply:
Okay, what if we just remove that sentence. Would that work? 
```suggestion
```

Reply:
Heds confirmed that removing that sentence addressed your concern.